### PR TITLE
LiveMap Widget Update

### PR DESCRIPTION
**Pull request by FatihKoz**
Added the ability to enable/disable flight table display below the map.

- Controller: Default/protected configuration for table is 'on'
- Blade: Checks the passed in config(table) variable to enable/disable the flights table below the map.

Also changed the aircraft.name to aircraft.registration in the flight table

### CODE REVIEW

File: app/Widgets/LiveMap.php [hunk 1]
```diff
@@ -14,6 +14,7 @@ class LiveMap extends Widget
     protected $config = [
         'height' => '800px',
         'width'  => '100%',
+        'table'  => 'on',
```
Comment (nabeelio) [inline app/Widgets/LiveMap.php:17]: Change this to a boolean

Comment (FatihKoz) [reply app/Widgets/LiveMap.php]: Changed

File: app/Widgets/LiveMap.php [hunk 2]
```diff
@@ -14,6 +14,7 @@ class LiveMap extends Widget
     protected $config = [
         'height' => '800px',
         'width'  => '100%',
+        'table'  => 1,
```
Comment (nabeelio) [inline app/Widgets/LiveMap.php:17]: Use `true`

File: resources/views/layouts/default/widgets/live_map.blade.php [hunk 3]
```diff
@@ -64,6 +64,7 @@
   </div>
 </div>
 
+@if($config['table'] == 'on')
```
Comment (nabeelio) [inline resources/views/layouts/default/widgets/live_map.blade.php:67]: `@if($config['table'] === true)`

Comment (FatihKoz) [reply resources/views/layouts/default/widgets/live_map.blade.php]: Why ? Specially where everything else in the config part was pure strings, what was the reason for having the boolen for the table display part.

I do understand making all in same type where possible without breaking the code flow but for this nothing was wrong by having it as string. This is just your personal wish i suppose while my personal wish was having it as a string like others :) It works either way and you are free to edit any pr's I sent, you do not need to wait me for the edits you think are needed.

**Pull request merged**